src/pineko/fonll.py

```diff
@@ -176,16 +188,33 @@ def produce_combined_fk(
     "FONLL-FFNS",
     "FONLL-FFNS",
 ]
+MIXED_NFFF_LIST = [3, 3, 4, 4, 4, 5, 5, 5]
+MIXED_PARTS_LIST = [
+    "full",
+    "full",
+    "massless",
+    "massive",
+    "full",
+    "massless",
+    "massive",
+]
 
-NFFF_LIST = [3, 3, 4, 4, 4, 5, 5, 5]
-PARTS_LIST = ["full", "full", "massless", "massive", "full", "massless", "massive"]
-FNS_BASE_PTO = {"FONLL-A": 1, "FONLL-B": 1, "FONLL-C": 2, "FONLL-D": 2, "FONLL-E": 3}
-MIXED_ORDER_FNS = ["FONLL-B", "FONLL-D"]
+# plain FONLL schemes
+FNS_LIST = [
+    "FONLL-FFNS",
+    "FONLL-FFN0",
+    "FONLL-FFNS",
+    "FONLL-FFN0",
+    "FONLL-FFNS",
+]
+NFFF_LIST = [3, 3, 4, 4, 5]
```

Comment:
@andreab1997 this makes more sense to me, e.g..

But I believe we can even generate this.